.github/workflows/lokalise.yml

```diff
@@ -0,0 +1,24 @@
+name: i18n
```

Comment:
what drove pulling this one out to its own file?

Reply:
As discussed offline, I want to avoid any skipped checks on PRs. Green checks only. As this check only runs when there is a merge to develop, I pulled it out into its own file so it would have its own `on` rules.